### PR TITLE
Roman Conversion In Python

### PR DESCRIPTION
**Pull request by prateeksharma21**

### CODE REVIEW

**Comment by jrg94** [comment]
Cool stuff! Since this is a new program to the repo, we'll need to outline some rules in our contributing doc. Could you summarize what your program does and add it to the [contributing doc](https://github.com/TheRenegadeCoder/sample-programs/blob/master/CONTRIBUTING.md)?

**Comment by prateeksharma21** [comment]
ok

File: archive/p/python/Roman Conversion.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                res = res + s2 - s1 
+                i = i + 2
+        else: 
+            res = res + s1 
+            i = i + 1
+  
+    return res 
+  
+# Driver code
+a=input("Enter Roman Number:")
```
Comment (jrg94) [inline archive/p/python/Roman Conversion.py]: This looks good! I have two requests:

1. Can we wrap the driver code in main?
2. Can we accept the Roman numeral over the command line instead of through `input`?

Comment (jrg94) [reply archive/p/python/Roman Conversion.py]: Also, can you rename the file to: `roman-numeral.py`

File: archive/p/python/roman-numeral.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                res = res + s2 - s1 
+                i = i + 2
+        else: 
+            res = res + s1 
+            i = i + 1
+  
+    return res 
+  
+def main():
+    a=sys.argv[1]
```
Comment (jrg94) [inline archive/p/python/roman-numeral.py]: Awesome! Two more comments:

1. Can we verify that there is a value at `sys.argv[1]`? Like `if len(sys.argv) > 1`.
2. Also, can we use a better variable name here? Something like `roman_numeral`.

**Comment by jrg94** [comment]
Thanks for being a good sport, @prateeksharma21! We appreciate your contribution. 😄

**Comment by prateeksharma21** [comment]
Sir , What is the problem now

**Comment by jrg94** [comment]
No problems! Just hoping you would add the Roman Numeral program to the Python README. I can do it if not.

**Comment by prateeksharma21** [comment]
Sir please do that

**Comment by jrg94** [comment]
No problem! I'll merge then.

**Comment by prateeksharma21** [comment]
Thanks Sir
